README.md

```diff
@@ -34,30 +52,79 @@ See the root `mise` config file at `home/vscode/.config/mise/config.toml` for al
 
 ## Using `mise` to Manage Go Versions
 
-1. Copy the `.mise.toml` file from the root of this repo to your projects repo root and modify it as needed.
-1. The provided `.devcontainer` will automatclly call `mise install` to install the custom versions of the applications.
-1. After the container is started and you shell into it you may need to call `mise use golang@<version>` to switch to the new version.  The included `help` explains how to do this in more detail and other `mise` commands you can use.
+1. Copy the `.mise.toml` file from the root of this repo to your project's root and modify it as needed.
+1. The provided `.devcontainer` will automatically call `mise install` to install the custom versions of the applications.
+1. After the container is started and you exec into it, `mise` will automatically install the app versions listed in the `.mise.toml` file.
+1. Other applications can also be installed by editing the `.mise.toml` file.  
```

Comment:
Numbered steps were not incremented here.

Reply:
They are not supposed to be.  This is Markdown best practice.  The downstrem instance automatically numbers them correctly.  This makes adding new items in random places and moving items around the list easier.